src/python/export_ghostfragment.cpp

```diff
@@ -14,11 +14,40 @@
  * limitations under the License.
  */
 
+#include "ghostfragment/property_types/fragmenting/capped_fragments.hpp"
+#include "ghostfragment/property_types/fragmenting/fragment_weights.hpp"
+#include "ghostfragment/property_types/fragmenting/fragmented_chemical_system.hpp"
+#include "ghostfragment/property_types/fragmenting/fragmented_nuclei.hpp"
+#include "ghostfragment/property_types/fragmenting/intersections.hpp"
+#include "ghostfragment/property_types/fragmenting/nuclear_graph_to_fragments.hpp"
+#include "ghostfragment/property_types/topology/broken_bonds.hpp"
+#include "ghostfragment/property_types/topology/connectivity_table.hpp"
+#include "ghostfragment/property_types/topology/nuclear_graph.hpp"
+#include "property_type/macros.hpp"
 #include <ghostfragment/ghostfragment.hpp>
 #include <pluginplay/pluginplay.hpp>
 
-namespace ghostfragment {
+namespace ghostfragment::pt {
```

Comment:
This changes the namespace the ghostfragment python module is extracted into. We want it to be extracted into the `ghostfragment` namespace, not `ghostfragment::pt`. 

```suggestion
namespace ghostfragment {
```